build_resolvers/lib/src/build_asset_uri_resolver.dart

```diff
@@ -130,16 +143,18 @@ class BuildAssetUriResolver extends UriResolver {
 
   @override
   Source resolveAbsolute(Uri uri, [Uri actualUri]) {
-    final cachedId = lookupAsset(uri);
-    if (cachedId == null) return null;
-    return resourceProvider
-        .getFile(assetPath(cachedId))
-        .createSource(cachedId.uri);
+    final assetId = parseAsset(uri);
+    if (assetId == null) {
```

Comment:
nit: for this one I would just do a one liner `if (assetId == null) return null;`